test-apps/ui-test-app/src/frontend/appui/frontstages/LocalFileStage.tsx

```diff
@@ -3,11 +3,11 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import * as React from "react";
-import { Id64String, ProcessDetector } from "@bentley/bentleyjs-core";
+import { Id64String } from "@bentley/bentleyjs-core";
 import { ViewDefinitionProps } from "@bentley/imodeljs-common";
 import { IModelConnection } from "@bentley/imodeljs-frontend";
 
-import { ElectronApp } from "@bentley/electron-manager/lib/ElectronFrontend";
+import { ElectronApp } from "@bentley/electron-manager/lib/ElectronFrontend"; // NEEDSWORK: No exporting barrel file in @bentley/electron-manager

```

Comment:
this is intentional because that package has both frontend and backend code in it. So there can't be one barrel file: there's two `lib/ElectronFrontend` and `lib/ElectronBackend` The alternative would be to make 2 packages. But if you're writing an Electron app you'd always need both and they'd always need to be the same version, so I deemed this preferable.